movie/src/components/MovieList/MovieList.css

```diff
@@ -1,18 +1,81 @@
 .movie {
   width: 100%;
+  box-sizing: border-box;
+}
+.navHight {
+  width: 100%;
+  background-color: rgb(3, 25, 96);
 }
 
-.allContent {
-  width: 70%;
+.topIntro {
+  width: 80%;
   margin: auto;
+  font-weight: bold;
+  margin-top: 20px;
 }
 
-.navHight {
-  background-color: rgb(3, 25, 96);
-  height: fit-content;
-  display: flex;
-  align-items: center;
+h3 {
```

Comment:
please use a class name for this

Reply:
Yep, I got your purpose, it is better If I set class to each html tag, because I can use "h3" many time on different page.